recipes/ezc3d/all/conandata.yml

```diff
@@ -1,8 +1,14 @@
 sources:
+  "1.3.8":
+    url: "https://github.com/pyomeca/ezc3d/archive/Release_1.3.8.tar.gz"
+    sha256: "888cb191c7939b31891e93356aa5bfcf21edb8785d3b5d4d9ec215984bc8b83e"
   "1.3.7":
     url: "https://github.com/pyomeca/ezc3d/archive/Release_1.3.7.tar.gz"
     sha256: "17cd3d4e20840710d58b8987c01ef0f880bc0c063ab52e0cb86ba41665ec33a0"
 patches:
+  "1.3.8":
+    - patch_file: "patches/fix-export-symbols.patch"
+      base_path: "source_subfolder"
```

Comment:
```suggestion
```

Reply:
this patch was submitted to upstream and is included in 1.3.8:
https://github.com/pyomeca/ezc3d/blob/Release_1.3.8/CMakeLists.txt
https://github.com/pyomeca/ezc3d/blob/Release_1.3.8/include/ezc3dConfig.h.in